### DISCUSSION TITLE
Outreachy Timeline and Support, and a Community Question

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/caMicroscope#840] [General]
Hi Everyone working towards outreachy! I think that the final application deadline is 2nd April, right?
As you know, this is caMicroscope's first year. As I know, I'm _very_ far behind on pull requests and community support.

That said, I wanted to ask people here a specific question: what will you need from us regarding outreachy specifically before that deadline? Please don't ask technical questions here, rather questions such as "What counts as a contribution for outreachy as far as caMicroscope is concerned?" (A: Anything on the suggested contribution list, even if we haven't reviewed/merged it yet, and even if we close it as duplicate ,as well as any other contributions even if they don't fit on the list).

We want to make sure that everyone has what they need here, and that we don't lose anything in the sheer volume (wow!) of contribution and community here. We've certainly underestimated all of the enthusiasm and support, and I appreciate you all!

Thank you again,
-Birm

**Comment by gbengaoluwadahunsi** [comment]
@birm ,

I  made over 10 pull requests within the first two weeks but since you told me my PRs are not separated and asked me to adjust so each PR is clarified, I closed the PR and made new ones that are clear cut, my question is would my PR be considered late or duplicated?

I also made tons of issues which thankfully other contributors picked up and worked on as contributions, will that also count for me?

Lastly, can you kindly check my PR and review/merge it?

Thank you

**Comment by barchakuz** [comment]
@birm 
 In my efforts to enhance project performance, I've identified critical issues requiring comprehensive restructuring of both frontend and backend code. These changes, detailed in issue summaries, promise significant performance improvements without altering core functionality. These restructuring tasks exceed the scope of typical pull requests but are crucial for project optimization. I seek clarity on whether these efforts qualify as valid contributions for Outreachy. Your guidance on integrating these contributions effectively would be invaluable. 
 Thank you for your consideration.

**Comment by We13b-MD** [comment]
@birm Thank you noted

**Comment by DIANAAJAMBO** [comment]
My question has been answered in your statement. Thanks @birm

**Comment by harshpreet14** [comment]
Thanks @birm

**Comment by nwanduka** [comment]
Thank you so much @birm for initiating this discussion and for the support so far. You've addressed one of my concerns about what counts as a valid contribution to caMicroscope during this contribution phase. Thanks again.

I do have (and I'm certain a good number of the contributors do too) a few questions regarding the final application.

1. We are required to share a project proposal timeline (here's [a template](https://docs.google.com/document/d/1Ji-eVZG4warXXrVE5f9urQerRD0SoP6yQkG3DSMwTWs/edit#heading=h.27qhk5r4oqg6) for more context). And are encouraged to work with our mentor to provide a timeline of the work we plan to accomplish on the project and what tasks we will finish at each step. How do we work this out? Maybe you could share your calendar with us so we know when you're available and draft our proposal timeline accordingly. (This is just a suggestion)

2. There's an optional field for "Community-specific questions" in the final application. We're to check with our mentor(s) to see if we need to provide any additional information that they may want to consider along with our application. Are there any questions you need us to answer here? Or should we leave it blank?

**Reply by Lochipi** [reply]
Hey @nwanduka. Did you get any response regarding the timeline?

**Comment by birm** [comment]
Outside of some specific circumstances, I'm unlikely to be able to review many individual proposals/schedules. I'm trying to get through the PR queue and I'm pretty sure I'll end up behind.
 (Sorry to everyone who submitted darkmode related PRs, I arbitrarily decided to "save those for last" 😢  Still do include them as contributions even if I don't get to them yet)

Re community questions, I'm not personally _too_ worried about anything specific, beyond the fact that I think a good proposal clearly answers "what are you proposing" "how will you do it" and "will you be able to do it". (it needn't be and probably shouldn't be structured this way literally)

What you're proposing should be reasonably straightforward. This should essentially be a more detailed/customized version of the project description, tailored for what you see as the need. It's ok if this is different from the project description as long as it's clear you've thought about the differences.

How you will do it is kind of the technical details. Somewhere between these two first questions would include your specific/achievable goals.

"will you be able to do it" is partially emergent. I think the two key things here are showing your competency, contributions, and/or relevant past projects, and the schedule seeming reasonable. It's ok to err a little safe with the schedule.

I hope this rambling message answers _some_ questions at least!

**Reply by Lochipi** [reply]
Yeah, sure, @birm, this helps.

**Reply by nwanduka** [reply]
Thank you so much @birm
